draft-packages/select/package.json

```diff
@@ -37,7 +37,8 @@
     "@kaizen/draft-form": "^3.1.6",
     "@types/classnames": "^2.2.10",
     "classnames": "^2.2.6",
-    "react-select": "^3.1.0"
+    "react-select": "^3.1.0",
+    "@types/react-select": "3.0.22"
```

Comment:
Was this related to the changes you made?

Reply:
Updating the draft button dependency seemed to remove the transitive dependency of @types/react-select; i think some other dependency of draft-button pulled in @types/react-select.
It wouldn't compile unless i added it.

Reply:
@jasononeil fyi
![image](https://user-images.githubusercontent.com/7651503/107609527-4d6e5200-6c93-11eb-857c-2b03664e951e.png)